src/jobs/models/jobManager.ts

```diff
@@ -0,0 +1,89 @@
+import { Logger } from '@map-colonies/js-logger';
+import { inject, injectable } from 'tsyringe';
+import { SERVICES } from '../../common/constants';
+
+import { ConnectionManager } from '../../DAL/connectionManager';
+import {
+  FindJobsResponse,
+  ICreateJobBody,
+  ICreateJobResponse,
+  IGetJobResponse,
+  IFindJobsRequest,
+  IJobsParams,
+  IUpdateJobRequest,
+  IJobsQuery,
+  IIsResettableResponse,
+  IResetJobRequest,
+} from '../../common/dataModels/jobs';
+import { JobRepository } from '../../DAL/repositories/jobRepository';
+import { EntityNotFound } from '../../common/errors';
+import { TransactionActions } from '../../DAL/repositories/transactionActions';
+
+@injectable()
+export class JobManager {
+  private repository?: JobRepository;
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger,
+    private readonly connectionManager: ConnectionManager,
+    private readonly transactionManager: TransactionActions
+  ) {}
+
+  public async findJobs(req: IFindJobsRequest): Promise<FindJobsResponse> {
+    const repo = await this.getRepository();
+    const res = await repo.findJobs(req);
+    return res;
+  }
+
+  public async createJob(req: ICreateJobBody): Promise<ICreateJobResponse> {
+    const repo = await this.getRepository();
+    this.logger.info('creating job');
```

Comment:
add parameters to log (resourceId, version and type) - or put the log inside the "repo.createJob(req)" method if its easier
If there is no "debug" log with all parameters - add it

Reply:
delete this.logger.info('creating job');